### PR TITLE
Add realio

### PR DESCRIPTION
**Pull request by hieuvubk**

### CODE REVIEW

**Comment by HeesungB** [comment]
@hieuvubk 
Failed to get response /status from rpc endpoint
Could you check endpoint?

**Comment by hieuvubk** [comment]
> 

Thank u I updated could u check again

**Comment by HeesungB** [comment]
@hieuvubk 
 - The token image should be named coinMinimalDenom. Could you change file name?
 - Image size should be 256x256px size. Could you resize image?
 - Could you remove `walletUrl`, `walletUrlForStaking` field?

**Comment by hieuvubk** [comment]
> * coinMinimalDenom

Thank so much addressed

**Comment by hoangdv2429** [comment]
@HeesungB we addressed all problems. Can you have a look ? Thank you.

**Comment by HeesungB** [comment]
@hieuvubk 
This chain's(`realionetwork_3301-1`) identifier is `realionetwork_3301`.
so, Could you rename to `realionetwork_3301` from `realio.json` and `images/realio` -> `images/realionetwork_3301`.

and Could you add chain image and change `chainSymbolImageUrl` field?

**Comment by hoangdv2429** [comment]
@HeesungB I changed chain's identifier and folder name, also image url paths. Can you have a look again. Thank you!

**Comment by hoangdv2429** [comment]
I see that `stakeCurrency` require to be a JSON object, but Realio actually have 2 stake-able tokens. @HeesungB what is your suggestion on this ? thank you so much!

**Comment by hoangdv2429** [comment]
@HeesungB I fix all the err when run the test on local. Can you review it now ? sorry to bother you at this hour. Thank you!

File: cosmos/realionetwork_3301.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
+{
+  "rpc": "https://realio-rpc.genznodes.dev",
+  "rest": "https://realio-api.genznodes.dev",
+  "nodeProvider": {
+    "name": "Genznodes",
+    "email": "genzbit@genznodes.dev",
+    "website": "https://genznodes.dev/"
+  },
+  "chainId": "realionetwork_3301-1",
+  "chainName": "Realioo",
```
Comment (rghunter) [inline cosmos/realionetwork_3301.json:10]: Should this be just `Realio`?

Comment (hoangdv2429) [reply cosmos/realionetwork_3301.json:10]: yes, it should.

**Review by rghunter — APPROVED** [review]

**Comment by hoangdv2429** [comment]
@HeesungB I think this PR is ready for merge now. Can you review it ? thank you.

**Pull request merged**